Tools/local-build-modules.R.in

```diff
@@ -0,0 +1,40 @@
+options(repos='https://cran.r-project.org')
+options(jaspRemoteCellarRedownload=FALSE)
+Sys.setenv(GITHUB_PAT="@GITHUB_PAT@")
+
+
+MODULES_DIR <- file.path("@PROJECT_BINARY_DIR@", "Modules")
+MODULES_SOURCE_ROOT <- file.path("@LOCAL_MODULES_ROOT@")
+MODULES <- "@LOCAL_MODULES_TO_BUILD@"
+
+modules <- sapply(strsplit(MODULES, ';'), trimws)
+modules <- file.path(MODULES_SOURCE_ROOT, modules)
+
+workdir <- file.path(MODULES_DIR, 'jaspLocalBuildWorkdir')
+toolLibpath <- file.path(workdir, 'toolLibpath')
+dir.create(toolLibpath, recursive=TRUE)
```

Comment:
```suggestion
if (!dir.exists(toolLibpath))
  dir.create(toolLibpath, recursive=TRUE)
```
otherwise each build shows
```
Warning message:
In dir.create(toolLibpath, recursive = TRUE) :
  '/.../github/jasp/jasp-desktop/build/Desktop_Qt_6_8_2-Debug/Modules/jaspLocalBuildWorkdir/toolLibpath' already exists
```